examples/kubernetes/access_points/specs/example.yaml

```diff
@@ -40,7 +40,7 @@ spec:
     - accesspoint=fsap-19f752f0068c22464
   csi:
     driver: efs.csi.aws.com
-    volumeHandle: fs-2e48aa59
+    volumeHandle: fs-e8a95a42
```

Comment:
should the example be demonstrating the new fs-e8a95a42::fs-e8a95a42 syntax?

Reply:
D'oh! I knew I forgot something.

Yes -- especially since we've said the old form was deprecated -- which I should also mention here.

/hold

Reply:
Done.

/hold cancel